communication/slack-guidelines.md

```diff
@@ -131,11 +131,9 @@ To add a channel, open a Pull Request (PR) updating the [slack-config].
   - Linking to resources such as the PR adding the subproject will speed in the
     validation and processing of the channel creation request.
   - Channels are **NOT**:
-    - Company specific; cloud providers are ok with product names as the channel.
-      Discourse will be about Kubernetes-related topics and not proprietary
-      information of the provider.
+    - Specific to a single company, commercial product, or commercial service
     - Private channels with the exception of: code of conduct matters, mentoring,
-      security/vulnerabilities, github management, or steering committee.
+      security/vulnerabilities, GitHub management, or steering committee.
```

Comment:
Just curious, but what's the reasoning behind having the steering committee channel be private? Or is this just stating that *some* steering committee conversations may be private due to discussions around licensing, legal, etc?

Reply:
I think it's just a realtime extension of the steering-private list, which is used when people need to bring up sensitive topics to steering: https://github.com/kubernetes/steering/blob/master/CONTRIBUTING.md#non-members

Reply:
ack, that's what I figured... thanks @castrojo! :)